src/Lime/Protocol/Client/Channel.ts

```diff
@@ -135,4 +172,13 @@ abstract class Channel implements MessageChannel, CommandChannel, NotificationCh
   }
 }
 
+class ClientError extends Error {

```

Comment:
This class is required? Just `throw Error` is not enought?

Reply:
This class was written in `blip-sdk-js`, so I moved this pattern to this file. Should we remove?

Reply:
Yes. Done 😃